playbooks/common/openshift-master/config.yml

```diff
@@ -337,12 +361,54 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
-- name: Configure master instances
-  hosts: oo_masters_to_config
+- name: Configure first master instance
```

Comment:
This has been reworked, the duplication is gone, the openshift_version is calculated for first master and set for all other hosts *very* early in the process via a playbook include. I believe this is required when you start installing containerized etcd envs and don't want to end up installing docker, then downgrading later.